src/components/scenes/CreateWalletSelectFiatScene.tsx

```diff
@@ -231,13 +249,17 @@ const CreateWalletSelectFiatComponent = (props: Props) => {
         />
         <ButtonsViewUi4
           primary={{
-            label: lstrings.title_create_wallets,
-            onPress: handleCreate
-          }}
-          secondary={{
-            label: lstrings.create_wallet_imports_title,
-            onPress: handleImport
+            label: isSplit ? lstrings.fragment_wallets_split_wallet : lstrings.title_create_wallets,
+            onPress: isSplit ? handleSplit : handleCreate
           }}
+          secondary={
+            isSplit
+              ? undefined
+              : {
+                  label: lstrings.create_wallet_imports_title,
+                  onPress: handleImport
+                }
+          }
```

Comment:
Since literally everything is different about the buttons view, it might make more sense to put the `isSplit` on the outside:

```tsx
isSplit ? 
  <ButtonsView primary={{ ...handleSplit }} /> :
  <ButtonsView primary={{ ...handleCreate }} secondary={{ ...handleImport }} />
```